pkg/cmd/config/config.go

```diff
@@ -18,9 +18,9 @@ func NewCmdConfig(f *cmdutil.Factory) *cobra.Command {
 	longDoc.WriteString("Display or change configuration settings for gh.\n\n")
 	longDoc.WriteString("Current respected settings:\n")
 	for _, co := range config.ConfigOptions() {
-		longDoc.WriteString(fmt.Sprintf("- %s: %s", co.Key, co.Description))
+		longDoc.WriteString(fmt.Sprintf("- `%s`: %s", co.Key, co.Description))
 		if co.DefaultValue != "" {
-			longDoc.WriteString(fmt.Sprintf(" (default: %q)", co.DefaultValue))
+			longDoc.WriteString(fmt.Sprintf(" (default: `%s`)", co.DefaultValue))
```

Comment:
@muzimuzhi : #8395 has been focused on addressing formatting of default values, which I'm hoping to work with @zsloane to get across the line.  The format there is a bit different than the backticks we are using for documentation.

That said I think we could easily ship the fencing of the settings without conflicting.

**What do you think?**